packages/dd-trace/test/debugger/devtools_client/utils.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
```

Comment:
Why are you moving this method out of integration-test directory? Is it expected to be used in non-integration tests?

Reply:
I need to access it in [an upcoming PR](https://github.com/DataDog/dd-trace-js/pull/5004) that adds benchmarks. I thought the test directory was a better place for it than the integration tests directory